static/js/services/enketo.js

```diff
@@ -254,12 +254,25 @@ angular.module('inboxServices').service('Enketo',
         });
     };
 
+    var attachContent = function(doc, content) {
+      doc._attachments = doc._attachments || {};
+      doc._attachments.content = {
+        content_type: 'application/xml',
+        data: btoa(content)
+      };
+    };
+
     var update = function(formInternalId, record, docId) {
       // update an existing doc.  For convenience, get the latest version
       // and then modify the content.  This will avoid most concurrent
       // edits, but is not ideal.
       return DB().get(docId).then(function(doc) {
-        doc.content = record;
+        // previously XML was stored in the content property
+        // TODO delete this and other "legacy" code support commited against
```

Comment:
@estellecomment I'm OK with it being lost honestly. There is nothing wrong with us supporting both formats, and there is no urgency or even need to drop support.

It's more of an indication that one of these formats is an "old" one (as opposed to both being currently valid). So maybe years from now if this gets refactored the developer can make a decision about dropping the old format, without being confused as to whether his refactor needs to support both.

^That is what I want to indicate anyway. But I also agree that you shouldn't have TODOs that don't resolve into tickets. What are your thoughts on how I can represent my thoughts. Just with a comment without the TODO maybe?